### PR TITLE
game_music_emu: bump to 0.6.3

### PR DESCRIPTION
**Pull request by theclue**
Bump is ok, everything is working

But to enable (optional) compressed file support Zlib is needed. Although i added to the requirements, CMake won't find it.

I was wondering if perhaps I have to manually tell CMake where Zlib headers and libraries are located or something...?

### CODE REVIEW

**Comment by Begasus** [comment]
Shouldn't there be a recipe included? 
Check with "cmake -L" if there is an option to set the path to the zlib headers/library?

**Comment by theclue** [comment]
> Shouldn't there be a recipe included?

I removed to make the recipe clean for merging

> Check with "cmake -L" if there is an option to set the path to the zlib headers/library?

Unfortunately not. I don't know how cmake works actually, so I was not able to dig into the source file actually.

**Comment by Begasus** [comment]
You are bumping a version, gme 0.6.2 won't pick up the 0.6.3 patchset at this stage, so you should update the recipe from 0.6.2 to 0.6.3?

**Comment by theclue** [comment]
I'm terribly sorry, I forgot to commit the 0.6.3 recipe -_- perhaps i was too tired to port, yesterday

**Comment by Begasus** [comment]
> I'm terribly sorry, I forgot to commit the 0.6.3 recipe -_- perhaps i was too tired to port, yesterday

Hence I close down in the evening ;)

**Comment by Begasus** [comment]
Can't seem to trace the whitespace here?

**Comment by theclue** [comment]
I see that CMake is trying to locate the Zlib library using `find_package()` function. As far as I know, this won't work as our Zlib has not the Cmake configuration file embedded.

File: media-libs/game-music-emu/game_music_emu-0.6.3.recipe [hunk 1]
```diff
@@ -15,30 +15,34 @@ HOMEPAGE="https://bitbucket.org/mpyne/game-music-emu"
...
+libVersionCompat="$portVersion compat >= ${portVersion%.*.*}"
+
 PROVIDES="
 	game_music_emu$secondaryArchSuffix = $portVersion
-	lib:libgme$secondaryArchSuffix = 0.6.2 compat >= 0
+	lib:libgme$secondaryArchSuffix = $libVersionCompat
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
+	lib:libzlib$secondaryArchSuffix
```
Comment (Begasus) [inline media-libs/game-music-emu/game_music_emu-0.6.3.recipe:35]: This should be:
```diff
- 	lib:libzlib$secondaryArchSuffix
+ 	lib:libz$secondaryArchSuffix
```

File: media-libs/game-music-emu/game_music_emu-0.6.3.recipe [hunk 2]
```diff
@@ -15,30 +15,34 @@ HOMEPAGE="https://bitbucket.org/mpyne/game-music-emu"
...
 
 PROVIDES_devel="
 	game_music_emu${secondaryArchSuffix}_devel = $portVersion
-	devel:libgme$secondaryArchSuffix = 0.6.2 compat >= 0
+	devel:libgme$secondaryArchSuffix = $libVersionCompat
 	"
 REQUIRES_devel="
 	game_music_emu$secondaryArchSuffix == $portVersion base
 	haiku${secondaryArchSuffix}_devel
+	devel:libzlib$secondaryArchSuffix
```
Comment (Begasus) [inline media-libs/game-music-emu/game_music_emu-0.6.3.recipe:45]: Line 44 and 45 should go to BUILD_REQUIRES, also devel:libz$secondaryArchSuffix

**Comment by theclue** [comment]
Done the requested changes, compressed files are now enabled :) Tnx for the support!

**Pull request merged**